.github/workflows/integration-install.yml

```diff
@@ -0,0 +1,18 @@
+name: Install Script Integration Test
+on:
+  pull_request:
+
+jobs:
+  install:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run Install Script 
+        run: ./scripts/install.sh
+      - name: Validate Command is Installed
+        run: |
+          if ! command -v draft &> /dev/null
```

Comment:
💡 Looks good and great to see check working : https://github.com/Azure/draft/actions/runs/3231609092/jobs/5291313431 

just minor fyi - I wonder how `draft` get set into `PATH` but all good as this clearly works.

Reply:
the github action runner PATH automatically includes the .local/bin directory 😄

Reply:
i can't find a doc for this, but it does work!